tests/Provider/FactoryTest.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DotTest\Rbac\Guard\Provider;
+
+use Dot\Rbac\Guard\Exception\RuntimeException;
+use Dot\Rbac\Guard\Provider\Factory;
+use Dot\Rbac\Guard\Provider\GuardsProviderInterface;
+use Dot\Rbac\Guard\Provider\GuardsProviderPluginManager;
+use PHPUnit\Framework\MockObject\Exception;
+use PHPUnit\Framework\TestCase;
+use Psr\Container\ContainerInterface;
+
+class FactoryTest extends TestCase
+{
+    protected Factory $subject;
+    protected ContainerInterface $container;
+
+    public function setUp(): void
+    {
+        $this->container = $this->createMock(ContainerInterface::class);
+
+        $this->subject = new Factory($this->container);
+    }
+
+    public function testCreateRuntimeException()
+    {
+        $this->expectException(RuntimeException::class);
+        $this->expectExceptionMessage('Guard provider type was not specified');
+        $this->subject->create([]);
+    }
+
+    /**
+     * @return void
+     * @throws Exception
+     */
+    public function testCreate()
```

Comment:
Please fix the following:
* add missing return type declaration
* remove line with `return void`